### PR TITLE
Eventhubs preview3 add new extension file

### PR DESCRIPTION
**Pull request by yunhaoling**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/eventhub/azure-eventhubs/azure/eventhub/extensions/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)  # type: ignore
```
Comment (lmazuel) [inline sdk/eventhub/azure-eventhubs/azure/eventhub/extensions/__init__.py]: @johanste this file ideally should be empty, unless we want easy "editable" mode and installed extensions at the same time

Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/azure/eventhub/extensions/__init__.py]: If this init file in azure-eventhubs is empty, should the one in the plugin under the same folder also be empty?

Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/azure/eventhub/extensions/__init__.py]: Updated

**Pull request merged**